src/sentry/static/sentry/app/components/deviceName.tsx

```diff
@@ -19,29 +15,32 @@ export async function loadDeviceListModule() {
   return import(/* webpackChunkName: "iOSDeviceList" */ 'ios-device-list');
 }
 
-export async function getDeviceName(model) {
-  const {default: iOSDeviceList} = await loadDeviceListModule();
-
-  return deviceNameMapper(model, iOSDeviceList);
-}
+type Props = {
+  children?: string;
+};
 
+type State = {
+  iOSDeviceList: any;
```

Comment:
It would be good if we added types for this library: https://github.com/pbakondy/ios-device-list#readme